pkg/config/template.go

```diff
@@ -553,52 +556,53 @@ const templateStringCrio = `[crio]
 
 const templateStringCrioRoot = `# Path to the "root directory". CRI-O stores all of its data, including
 # containers images, in this directory.
-root = "{{ .Root }}"
+{{ .Comment }}root = "{{ .Root }}"
 
 `
 
 const templateStringCrioRunroot = `# Path to the "run directory". CRI-O stores all of its state in this directory.
-runroot = "{{ .RunRoot }}"
+{{ .Comment }}runroot = "{{ .RunRoot }}"
 
 `
 
 const templateStringCrioStorageDriver = `# Storage driver used to manage the storage of images and containers. Please
 # refer to containers-storage.conf(5) to see all available storage drivers.
-storage_driver = "{{ .Storage }}"
+{{ .Comment }}storage_driver = "{{ .Storage }}"
 
 `
 
 const templateStringCrioStorageOption = `# List to pass options to the storage driver. Please refer to
 # containers-storage.conf(5) to see all available storage options.
-storage_option = [
-{{ range $opt := .StorageOptions }}{{ printf "\t%q,\n" $opt }}{{ end }}]
+
+{{ .Comment }}storage_option = [
+{{ range $opt := .StorageOptions }}{{ $.Comment }}{{ printf "\t%q,\n" $opt }}{{ end }}{{ $.Comment }}]
```

Comment:
will this $ get in the way of the Replace above?

Reply:
As part of kata-containers CI we generate the crio.conf file by calling `crio -d "" --log-level "debug" --cgroup-manager "systemd" --storage-driver "overlay" --storage-opt "overlay.override_kernel_check=1" config > crio.conf`, and this will generate
```
storage_option = [
#       "overlay.override_kernel_check=1",
# ]
```

That's the reason the kata-containers CI is failing.

I'd strongly suggest adding a test case covering something similar.